trajectory_msgs/JointTrajectory   waypoints
bool                              publish_path
bool                              publish_trajectory
bool                              plan_path
bool                              plan_trajectory
bool                              override_dynamic_constraints
float64[]                         velocity_constraints
float64[]                         acceleration_constraints
---
trajectory_msgs/JointTrajectory   trajectory
trajectory_msgs/JointTrajectory   path
bool                              success
float64                           path_length
float64[]                         roll
float64[]                         pitch